lib/ecto/adapters/sql.ex

```diff
@@ -128,6 +130,13 @@ defmodule Ecto.Adapters.SQL do
       ## Migration
 
       @doc false
+      def execute_ddl(repo, %View{}=definition, opts) do
```

Comment:
We will need to change `@conn.execute_ddl` contract then altogether. Something like:

```elixir
# TODO: Enforce the {sql, params} on Ecto 3.0
for entry <- List.wrap(@conn.execute_ddl(definition)) do
  case entry do
    {sql, params} -> Ecto.Adapters.SQL.query!(repo, sql, params, opts)
    sql -> Ecto.Adapters.SQL.query!(repo, sql, params, opts)
  end
end
```